src/identity/core/getName.ts

```diff
@@ -1,15 +1,58 @@
-import { publicClient } from '../../network/client';
+import { mainnet } from 'viem/chains';
 import type { GetName, GetNameReturnType } from '../types';
+import L2ResolverAbi from '../../abis/L2ResolverAbi';
+import {
+  resolverAddressesByChainId,
+  convertReverseNodeToBytes,
+} from '../../utils/ens';
```

Comment:
If an utility is used only in one module, let's keep those inside the `core` folder of that module.

Reply:
moved